UPGRADE-1.12.md

```diff
@@ -15,3 +15,29 @@ Service aliases
 
  * Deprecated the `Symfony\Bridge\Doctrine\RegistryInterface` and `Doctrine\Bundle\DoctrineBundle\Registry` service alias, use `Doctrine\Common\Persistence\ManagerRegistry` instead.
  * Deprecated the `Doctrine\Common\Persistence\ObjectManager` service alias, use `Doctrine\ORM\EntityManagerInterface` instead.
+
+UnitOfWork clear between each request
+-------------------------------------
```

Comment:
```suggestion
---------------------------------------
```

Reply:
That doesn't look right?

Reply:
Not unless you apply my first suggestion